package.json

```diff
@@ -14,6 +14,7 @@
     "@types/node": "^14.11.2",
     "@types/react": "^16.9.49",
     "@types/react-dom": "^16.9.8",
+    "clsx": "^1.1.1",
```

Comment:
I ended up not using this `clsx` should I remove from here or leave it?

Reply:
Hmm, I recommend removing it so our builds are slightly faster

Reply:
Alright I removed it